applications/StructuralMechanicsApplication/custom_elements/shell_thin_element_3D4N.cpp

```diff
@@ -1838,7 +1838,7 @@ namespace Kratos
 			for (int i = 0; i < 4; i++)
 			{
 				//eqn 5.2.29
-                                const Vector vec1 = MathUtils<double>::CrossProduct(data.r_cartesian[i], s_xi);
+                const Vector vec1 = MathUtils<double>::CrossProduct(data.r_cartesian[i], s_xi);

```

Comment:
this code should be (to my understanding)

     Vector vec1(3);
     MathUtils<double>::CrossProduct(vec1,s_xi,data.r_cartesian[i]); //changed the order to maintain the sign

Reply:
I am using [this function](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/utilities/math_utils.h#L827)
which is not deprecated in order to be consistent with the rest of the code

If I should use what you suggest then I have to change a lot